plugins/catalog-node/src/testUtils/catalogServiceMock.ts

```diff
@@ -21,7 +21,7 @@ import {
 } from '@backstage/backend-plugin-api';
 import { InMemoryCatalogClient } from '@backstage/catalog-client/testUtils';
 import { Entity } from '@backstage/catalog-model';
-import { catalogServiceRef } from '@backstage/plugin-catalog-node';
+import { catalogServiceRef } from '@backstage/plugin-catalog-node/alpha';
```

Comment:
@Rugvip - I've managed to use `catalogServiceMock` in the tests by just making this slight change without disturbing the public API. Once it's decided to transition to using the non-alpha, `catalogServiceRef`, the transition would reflect here as well. What do you think?

Reply:
Hmm, I think that we're probably gonna want to use the one from the root, I have a feeling that this is using the old API definition in the apiRef, rather than the new one, which is why this works.

There's a PR that andre put together for removing the old backend system from scaffolder which is going to be a breaking change anyways. I wonder if we can get that shipped and include the breaking change for the actions pretty quick as to unblock this work.

Reply:
@benjdlambert - could you please share that PR is that so I can follow?